### ISSUE TITLE
Node v18+ issue with SimpleLinearRegression() constructor

### ISSUE THREAD

**Issue by nick-hunter** [mljs/regression-simple-linear#11]
I recently updated my version of Node from 16.17.0 to 18.16.0 and ran into an issue with the constructor. I also tested this with v20.5.0 and still had the problem.

When using the example code in an ES Module, I'm getting the following error:

```
const regression = new SimpleLinearRegression(x, y);
                   ^

TypeError: SimpleLinearRegression is not a constructor
    at file:///home/hunten/ml-test/index.js:8:20
    at ModuleJob.run (node:internal/modules/esm/module_job:192:25)
    at async DefaultModuleLoader.import (node:internal/modules/esm/loader:228:24)
    at async loadESM (node:internal/process/esm_loader:40:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)
```

However this does work `const regression = new SimpleLinearRegression.default(x, y);`. Am I doing something wrong or is there an issue with the export?

**Comment by targos** [comment]
Hello, this wasn't intended and is probably caused by the migration to TypeScript. @santimirandarp could you check if it's possible to configure TS so it directly exports the default as before in CommonJS (`module.exports = TheClass` instead of `exports.default = TheClass`) ?

**Comment by santimirandarp** [comment]
I am searching, but it seems that the easy, and trouble-free way would be to use named exports, although it's a breaking change. wdyt @targos  ?

**Comment by targos** [comment]
@santimirandarp OK, let's make the breaking change (named export), and I'll publish a patch version that reverts the TS update in v2.

**Comment by santimirandarp** [comment]
Great, let me double test things to make sure (as much as i can) that there aren't other problems.

**Comment by targos** [comment]
This should be fixed. I published v2.0.5 from the v2.0.3 tag to override the bad version.
